account_manager/src/validator/exit.rs

```diff
@@ -165,6 +166,43 @@ async fn publish_voluntary_exit<E: EthSpec>(
             "Did not publish voluntary exit for validator {}. Please check that you entered the correct exit phrase.",
             keypair.pk
         );
+        return Ok(());
+    }
+
+    loop {
+        // Sleep for 1 slot duration and then check if voluntary exit was processed
+        // by checking the validator status.
+        // TODO: should we wait for longer to account for forks?
```

Comment:
TODO

Perhaps we just update the "Voluntary exit has been accepted..." text to say something along the lines of:

> "Voluntary exit has been accepted into the beacon chain, but not yet finalized. Finalization may take several minutes or longer. Before finalization there is a low probability that the exit may be reverted."

Reply:
Yep, this sounds perfect. Will make the changes.